xcodeproj/internal/xcode_targets.bzl

```diff
@@ -1,1024 +0,0 @@
-"""Module containing functions dealing with the `Target` DTO."""
```

Comment:
Is the intent to delete this file? I don't see this logic being moved somewhere else?

Reply:
No, something odd with my git-pile implementation. Fixing.